src/components/IntroLinear/StoryLine/StoryStepper.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import {Typography } from "@material-ui/core";
+
+import MobileStepper from "@material-ui/core/MobileStepper";
+
+import Button from "@material-ui/core/Button";
+
+
+const StoryStepper = ({ timesClicked, handleNextClick, handleBackClick }) => {
+
+    const [activeStep, setActiveStep] = React.useState(0);
+
+    const handleNext = () => {
+        setActiveStep((prevActiveStep) => prevActiveStep + 1);
+        handleNextClick();
+    };
+
+    const handleBack = () => {
+        setActiveStep((prevActiveStep) => prevActiveStep - 1);
+        handleBackClick();
+    };
+
+    if (timesClicked < 2) {
```

Comment:
Do we need this conditional? It looks like they both render the same thing

Reply:
Good catch, I think originally I had something planned for the conditional but decided against it